tests/Klasse.spec.ts

```diff
@@ -133,4 +133,52 @@ test.describe(`Testfälle für die Administration von Klassen: Umgebung: ${proce
       await expect(klasseCreationView.button_ZurueckErgebnisliste).toBeVisible();
     });
   });
+
+  test("Jede Klasse hat eine Dienststellennummer neben dem Klassennamen (mit Paginierung)", { tag: [LONG, SHORT, STAGE] }, async ({ page }) => {
+    const startseite = new StartPage(page);
+    const menue = new MenuPage(page);
+    const klasseManagementView = new KlasseManagementViewPage(page);
+
+    await test.step(`Klassenverwaltung öffnen und prüfen, dass jede Klasse eine Dienststellennummer hat`, async () => {
+        // Navigate to Klassenverwaltung
+        await startseite.card_item_schulportal_administration.click();
+        await menue.menueItem_AlleKlassenAnzeigen.click();
+
+        // Wait until the table is visible
+        await expect(klasseManagementView.text_h2_Klassenverwaltung).toHaveText("Klassenverwaltung");
+
+        // Define the selector for the 'Next' button
+        const nextPageButton = page.locator('.v-pagination__next button:not(.v-btn--disabled)');
+
+        // Loop over each page
+        let hasNextPage = true;
+        while (hasNextPage) {
```

Comment:
I recommand not to use conditionals in ui testautomation (if possible). 
The loop doesn't work. The test will fail when the timeout is reached in 60s. The list of classes can have thousends of elements. 
Maybe its better to check the first 100 entries and the last 100 entries. Then you have a defined amount of classes to test.
Do something like that: 
1) Show first 100 entries
await page.getByTestId('klasse-table').getByRole('combobox').locator('i').click();
await page.getByRole('option', { name: '100' }).click();
2) check the entries
3) Go to the last page
await page.getByLabel('Letzte Seite').click();
4)  check the entries

Reply:
I check only for the first and last 100 entries in the table.